templates/tfengine/components/cicd/README.md

```diff
@@ -1,21 +1,134 @@
-# Continuous Integration (CI) and Continuous Deployment (CD) component
+# Continuous integration (CI) and continuous deployment (CD)
 
-This directory contains the Terraform configs for the Continuous Integration (CI)
-and Continuous Deployment (CD) component.
+This directory defines the resources that need to be deployed manually to set up
+CICD pipelines of Terraform configs.
 
-* The [manual/](./manual) directory contains the Terraform resources that need to
-  be deployed manually to provision the initial CI/CD resources.
+The CI and CD pipelines use
+[Google Cloud Build](https://cloud.google.com/cloud-build) and
+[Cloud Build triggers](https://cloud.google.com/cloud-build/docs/automating-builds/create-manage-triggers)
+to detect changes in the repo, trigger builds, and run the workloads.
 
-* The [auto/](./auto) directory contains the Terraform resouces for CI/CD purposes
```

Comment:
Add a section in templates/tfengine/components/cicd/README.md to cover this part of the content even now we don't have the auto/ folder.